Login modifications on tracker to work with training application

Ezmeyers opened the pull request:
Here are the changes to tracker-capture to allow automatic login with training application, will create a pull request for training app shortly. thanks!

sunbiz reviewed (changes requested):
I don't think we should ~~include~~ modify the gradlew, gradle-wrapper.jar, gradle-wrapper.properties and other files. ~~Can you please remove those files from the commit, or add them to .gitignore, so that they don't get committed again.~~

Ezmeyers commented:
The tracker-capture hosted by DHIS2 has these included, do you still want me to remove? If so, no problem

sunbiz commented on app/src/main/AndroidManifest.xml:
```diff
@@ -116,15 +115,25 @@
                 android:resource="@xml/file_paths" />
         </provider>
+
+        <receiver
```
Only this change is required. All the other changes are not appropriate

sunbiz commented on app/src/main/java/org/hisp/dhis/android/trackercapture/MainActivity.java:
```diff
@@ -35,20 +35,26 @@
 import android.content.Intent;
```
No change is required to MainActivity.java

sunbiz commented on app/gradlew:
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```
No changes required to app/gradlew

sunbiz commented on app/gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -0,0 +1,6 @@
+#Thu Nov 30 01:08:05 EST 2017
```
No changes required to app/gradle/wrapper/gradle-wrapper.properties

sunbiz commented on app/gradlew.bat:
```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```
No changes are required to app/gradlew.bat

sunbiz commented on app/proguard-rules.pro:
```diff
@@ -15,3 +15,5 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
```
No changes required to app/proguard-rules.pro

Ezmeyers committed:
Automatic login to training app

added login to training app from tracker-capture sidebar and through broadcast receiver on training startup, amended readme, added login flag hotfix for errors logging in from tracker
